### DISCUSSION TITLE
where to get grub/grub.0

### DISCUSSION THREAD

**Discussion by wanghaiqing2015** [cobbler/cobbler#3346] [Q&A] [answered]
cobbler 3.1.2, where to get grub/grub.0.

![11111](https://user-images.githubusercontent.com/11815186/86792553-33391980-c09d-11ea-891d-8724b9fd314e.png)

```
cat /etc/dhcp/dhcpd.conf
     class "pxeclients" {
          match if substring (option vendor-class-identifier, 0, 9) = "PXEClient";

          # Legacy
          if option system-arch = 00:00 {
              filename "grub/grub.0";
          }
```

**Comment by wanghaiqing2015** [comment]
Run "cobbler get loaders" without downloading this file.

**Comment by wanghaiqing2015** [comment]
Legacy boot

**Comment by im-jinxinwang** [comment]
> Legacy boot

Can you email me to inform your WeChat account? Feel like we are from the same place

**Comment by wanghaiqing2015** [comment]
> > Legacy boot
> 
> Can you email me to inform your WeChat account? Feel like we are from the same place

Where is your email ？

**Comment by watologo1** [comment, marked as answer]
get loaders
is rather unsecure and should not be used.

Execute:
./scripts/mkgrub.sh

It could be that for your distro some adjusting is needed. Let me know if it does not work as expected.

**Comment by wanghaiqing2015** [comment]
thanks

![222](https://user-images.githubusercontent.com/11815186/86893210-dccaea00-c133-11ea-844e-25b144ce1095.png)

**Comment by im-jinxinwang** [comment]
> > > Legacy boot
> > 
> > 
> > Can you email me to inform your WeChat account? Feel like we are from the same place
> 
> Where is your email ？

I can see the email address on my git homepage
